py/dml/ctree.py

```diff
@@ -356,6 +358,28 @@ def toc(self):
 
 mkLabel = Label
 
+class UnrolledLoop(Statement):
+    @auto_init
+    def __init__(self, site, substatements, break_id):
+        assert isinstance(substatements, list)
+
+    def toc(self):
+        out('{\n', postindent = 1)
+        self.toc_inline()
+        out('}\n', preindent = -1)
+
+    def toc_inline(self):
+        for substatement in self.substatements:
+            substatement.toc()
+        if self.break_id is not None:
+            out(f'_unrolledbreak{self.break_id}: UNUSED;\n')
```

Comment:
is UNUSED needed? (I'm just curious; if you are unsure and just added this defensively then that's ok)

Reply:
I mentioned it in an earlier comment, but yeah, it's done defensively. I'm slightly concerned some code elimination shenanigans might drop a break statement, even though `stmt_break` is called.